src/app/lib/layoutGrid/index.js

```diff
@@ -129,13 +129,13 @@ export const layoutGridItemMedium = css`
 
   grid-column: 1 / span 6;
   @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
-    padding: 0 ${props => (props.padding && props.padding.group2) || 'auto'};
+    ${({ padding: { group2 } = {} }) => (group2 ? `padding: 0 ${group2}` : '')};
```

Comment:
I think I prefer `${({ padding = {} }) => (padding.group2 && `padding: 0 ${padding.group2}`)};`, but I'm happy either way if code maintainers are swayed one way or the other.

Reply:
I prefer ```${({ padding = {} }) => (padding.group2 && `padding: 0 ${padding.group2}`)};``` too, thoughts @jroebu14 ?

Reply:
Yeh I'm not a fan of deep destructuring either. That looks good. However the ternary is needed because of the string interpolation. Otherwise you'd end up with `undefined` in the CSS if `group2` is undefined.

Reply:
Does styled-components not filter the undefined out for us?

Reply:
Oh does it? Feels weird allowing something like that in the code though.

Reply:
@dr3 I refactored the destructuring however I didn't like the idea of relying on styled-components to remove the `undefined` string so I stuck with the ternary. I wouldn't want to get used to doing that in case it catches me out in something outside of styled-components. If you feel strongly about it I can remove the ternary though 🙂